### DISCUSSION TITLE
Possibility to hide submenu

### DISCUSSION THREAD

**Discussion by cosmincelea** [directus/directus#21039] [Feature Requests]
### Summary

It would be really nice if it would be possible to hide the submenu. Just a simple minify button that hides the submenu.

### Motivation

It really helps on smaller screens to save the space of the submenu and visualise only the information that is relevant. 

### Detailed Design

Simple minification button in the top right corner, that hides the submenu. It can be shown again when clicking on the main menu item again.

### Must Have:
- A Possibility to hide the submenu
- B Possibility to show it again

### Should Have:
- C Simple design

### Could Have:
- D A shortcut option

### Won't Have:
- E

### Drawbacks

I think it would be a fast feature to implement

### Alternatives

No alternatives

### Adoption Strategy

No alternatives

**Comment by ched-dev** [comment]
What submenu are you referring to? Can you please include screenshots?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
